news.html

```diff
@@ -22,13 +22,25 @@
         </div>
 
         <h1>NEWS</h1>
-        <div class="enclosed-box infobox">
-            <h2>Club Meeting Friday 1/8</h2>
+        <!-- Put news in reverse chronological order. -->
+
+        <div class="post infobox">
+            <h1 class="post-title">Club Meeting Friday 1/8</h1>
```

Comment:
I based it on the format used by the [Bootstrap blog](http://blog.getbootstrap.com/).